pom.xml

```diff
@@ -296,8 +245,7 @@
         <plugin>
           <groupId>org.apache.felix</groupId>
           <artifactId>maven-bundle-plugin</artifactId>
-          <!-- keep this version as upgrade break manifest -->
-          <version>4.0.0</version>
+          <version>5.1.4</version>
```

Comment:
need to test/review manifest generation as upgrade necessary for java 17 but might break some stuff